packages/ui/src/components/auth-page/page-wrapper.tsx

```diff
@@ -14,31 +15,27 @@ import {
  * 3. FooterWrapper
  *
  */
-export function PageWrapper({
-  pageTitle,
+export function PageWrapper({ children }: { children: React.ReactNode }) {
+  return (
+    <div className="h-full w-full">
+      <section className="md:max-w-md md:mx-auto md:pt-10 py-5">
+        <Card>{children}</Card>
+      </section>
+    </div>
+  );
+}
+
+export function FormHeader({
   cardTitle,
   cardDescription,
-  children,
 }: {
-  pageTitle: string;
-  children: React.ReactNode;
   cardTitle: string;
-  cardDescription?: string;
+  cardDescription?: React.ReactNode;
```

Comment:
```suggestion
  cardDescription?: React.ReactNode;
```
cardDescription name suggests that it is a string, can a more meaningful name be used for the same. e.g. cardDescriptionComp ?